sos/policies/auth/__init__.py

```diff
@@ -67,7 +67,8 @@ def _request_device_code(self):
             res = requests.post(
                 self.client_identifier_url,
                 data=data,
-                headers=headers)
+                headers=headers,
+                timeout=300)
```

Comment:
I am unsure what the right timeout value can be; picked 300 to be a conservative value. Can increase/decrease depending on feedback.

Reply:
We have a `TIMEOUT_DEFAULT = 300` in `sos/utilities.py`, maybe we can utilise that value? This is already set to 300

Reply:
300 or TIMEOUT_DEFAULT should be enough as a starting point for RH auths

Reply:
ack, used TIMEOUT_DEFAULT from sos/utilities.py